Tests/SwerverTests/RouterSpec/RouterSpec.swift

```diff
@@ -6,121 +6,212 @@ import Foundation
 class RouterSpec: QuickSpec {
```

Comment:
I would take a look at all of your specs here. You use `expected` a lot of times where you mean `actual`. There are also a couple of places where I'm not sure the extra variable improves readability.

Reply:
Also, your `Router` might be a bit over-tested at this point. Now that you are injecting the routes, there's not much difference between routing a valid HEAD request or routing a valid GET request. I would see which of these tests are actually testing the router, and which are testing the actions that no longer live in the router.

Reply:
will look into that

Reply:
fixed